entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+PASSPHRASE=$(/usr/local/bin/passphrase.sh)
+echo $PASSPHRASE
+AKAMAI_SECRET=$(echo $AKAMAI_SECRET_KEY|openssl base64 -d|openssl enc -d -pbkdf2 -k $(echo $PASSPHRASE))
```

Comment:
Have you verified that the env variable is accessible in Theia? The reason I'm asking is because I'm not sure off the top of my head how env variables and exec behave together, do they need to be exported or not.

Reply:
@bezidejni: Yep, I ran this by passing in variables at the command line with `-e`, which we mimic with the env vars in Image Settings.

Reply:
Merge away then!